types/wrapper.go

```diff
@@ -28,10 +29,13 @@ type rawWrapper struct {
 	Value      *json.RawMessage `json:"spec" yaml:"spec"`
 }
 
+// PackageMap contains a list of packages with their Resource Resolver func
 var packageMap = map[string]func(string) (Resource, error){
 	"core/v2": v2.ResolveResource,
 }
 
+var packageMapMu = &sync.Mutex{}
```

Comment:
Need to make use of `packageMapMu` in the `UnmarshalJSON` method of `Wrapper` as well